x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/fulltext/Match.java

```diff
@@ -136,7 +137,7 @@ public Match(Source source, Expression field, Expression matchQuery, QueryBuilde
         this.field = field;
     }
 
-    private static Match readFrom(StreamInput in) throws IOException {
+    protected static Match readFrom(StreamInput in) throws IOException {
```

Comment:
I don't think that this static method needs to be protected? It's not accessed from the subclass, right?

Reply:
Correct, thank you! Fixed in 346bf9d6e8779910ed163961c76dbc195ed81858